LOAN/LoanTypes/SecuredLoans.rdf

```diff
@@ -68,17 +75,17 @@
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-loan-typ-sec;GuaranteedLoan">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-sec;SecuredLoan"/>
+		<rdfs:subClassOf rdf:resource="&fibo-loan-ln-ln;SecuredLoan"/>
```

Comment:
(thought I made this comment already with a reference)
I think this is wrong: secured and guaranteed are orthogonal.